godtools-renderer/src/main/java/org/keynote/godtools/renderer/crureader/bo/GPage/GTitle.java

```diff
@@ -0,0 +1,242 @@
+                                                                          autoScaleTextView.setTextColor(RenderSingleton.getInstance().getPositionGlobalColorAsInt(position));
+                                                                          peekPanel.updateBaseAttributes(autoScaleTextView);
+                                                                          final PopupWindow pw = new PopupWindow(inflatedView);
+                                                                          if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP_MR1) {
+                                                                              pw.setAttachedInDecor(true);
+                                                                          }
+                                                                          pw.setTouchInterceptor(new View.OnTouchListener() {
+                                                                              @Override
+                                                                              public boolean onTouch(View v, MotionEvent event) {
+
+                                                                                  pw.dismiss();
+                                                                                  new Thread()
+                                                                                  {
+                                                                                      public void run()
+                                                                                      {
+                                                                                          super.run();
+                                                                                          try {
+                                                                                              sleep(15);
+                                                                                              ((Activity)context).runOnUiThread(new Runnable() {
+                                                                                                  @Override
+                                                                                                  public void run() {
+                                                                                                      view.setClickable(true);
+                                                                                                  }
+                                                                                              });
+                                                                                          } catch (InterruptedException e) {
+
+
+                                                                                          }
+                                                                                      }
+                                                                                  }.start();
```

Comment:
why not use:
```
v.postDelayed(new Runnable() {
    view.setClickable(true);
}, 15);
```

instead of manually creating a thread to trigger something on the UI thread?